ui/app/styles/core/table.scss

```diff
@@ -86,6 +86,30 @@
       overflow: hidden;
       text-overflow: ellipsis;
       white-space: nowrap;
+
+      &.is-hover-overflowable {
+        position: relative;
+
+        .hover-overflow {
+          display: none;
+          position: absolute;
+          top: 0;
+          left: 0;
+          padding: calc(1.25em - 1px) 1.5em;
+        }
+
+        &:hover {
+          color: transparent;
+          overflow: visible;
+
+          .hover-overflow {
+            color: $text;
+            display: block;
+            background-color: rgba(255, 255, 255, 0.9);
+            border-right: 1px solid $grey-blue;
```

Comment:
As discussed offline, having some sort of "shadow" (actually a gradient) will add depth and make the obscuring effect more obvious. Probably best implemented with an `::after` and a `background: linear-gradient(...)` since there is that weird transparency + border-image issue.